ChangeLog.d/update-error-codes.txt

```diff
@@ -0,0 +1,5 @@
+Removals
```

Comment:
Actually it's not just a removal, but rather a change. So I'm undecided between "API changes" or simply "Changes¨, but I have a smaller preference for the former since it's a change that affects the return value of some APIs. Maybe other reviewers have different opinion though.

Reply:
I am happy to go with API change